xml/System/Environment+SpecialFolder.xml

```diff
@@ -539,7 +539,7 @@
       </ReturnValue>
       <MemberValue>23</MemberValue>
       <Docs>
-        <summary>A folder for components that are shared across applications. This special folder is valid only for Windows NT, Windows 2000, and Windows XP systems. Added in the .NET Framework 4.</summary>
```

Comment:
Good catch here. The old sentence made it sound like we only supported this up to XP, but I think they meant to say it's supported in Windows NT and above, because we are [still supporting it](https://github.com/dotnet/runtime/search?q=extension%3Acs+CommonPrograms).